src/main/java/graphql/schema/GraphQLFieldDefinition.java

```diff
@@ -184,13 +162,6 @@ public Builder deprecate(String deprecationReason) {
         }
 
         public GraphQLFieldDefinition build() {
```

Comment:
> @dminkovsky has raised concern about the fetcher now being required by the field definition interface. ... Required how?

Thanks @IamCornholio. To clarify: we were thinking that perhaps there should be no default `DataFetcher`. If there is no default `DataFetcher`, and the user does not specify a data fetcher, then building a field will cause an exception. That's fine if you're specifying a field for a `GraphQLObjectType`. You want every field to have a `DataFetcher`. But if you're specifying a field for a `GraphQLInterfaceType`, then you want to be able to omit the data fetchers.